### PR TITLE
Office bearers

### PR DESCRIPTION
**Pull request by killbotXD**
Created 2 files

1. officeBearers.vue - contains the list of all office bearers
2. officeBearerCard.vue - component to show individual cards of office bearers

Future tasks:
1. Add social links of all office bearers if they exist.
2. Fix the top bar
3. Use Graphql

Screenshot from 2019-12-13 15-28-27
Screenshot from 2019-12-13 15-28-37
Screenshot_2019-12-13-15-28-44-242_com android chrome
Screenshot_2019-12-13-15-28-46-986_com android chrome

### CODE REVIEW

File: src/components/OfficeBearerCard.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          v-img(src="https://students.iitj.ac.in/static/assets/others/cover2.svg" :height="avatarSize")
+          v-card-text.pt-0.text-center
+            v-avatar.elevation-4.ma-2(:size="avatarSize" :style="{'margin-top': `-${avatarSize/2}px !important`}")
+              v-img(src="https://cdn2.iconfinder.com/data/icons/people-80/96/Picture1-512.png")
+            h2(class="black--text").font-weight-bold.mt-5 Anugrah Shukla
+            p.font-weight-regular.mt-2 General Secretary
+          v-card-actions
+            v-row(justify="center")
+              v-btn(@click="cardPage = 2" text)
+                v-icon mdi-refresh
```
Comment (ajatprabha) [inline src/components/OfficeBearerCard.vue]:
```
              v-btn(@click="cardPage = 2" text) Details
                v-icon(right) mdi-arrow-right
```

File: src/components/OfficeBearerCard.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+              v-row.justify-center
+                div
+                  v-row
+                    v-icon  mdi-email
+                    v-text.ma-2.pa-2 kumar.14@iitj.ac.in
+              v-row.justify-center.align-end.pa-0.ma-0.fh
+                v-divider
+              v-row.justify-center.mt-2.pa-0
+                v-btn(@click="cardPage = 1" text)
+                  v-icon mdi-refresh
```
Comment (ajatprabha) [inline src/components/OfficeBearerCard.vue]:
```
                  v-icon(left) mdi-arrow-left
                  | Back
```

File: src/components/OfficeBearerCard.vue [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          v-row
+            v-col
+              v-row
+                v-card-text.justify-center.text-center
+                  h2.font-weight-medium.mb-5  About Anugrah
+              v-row
+                v-divider
+              v-row.justify-center
+                div.mt-4
+                  a(href="https://students.iitj.ac.in/office-bearers/tel:9610942435") +91-9610942435
```
Comment (ajatprabha) [inline src/components/OfficeBearerCard.vue]: a(href="tel:9610XXXXXX") +91-9610XXXXXX

File: src/components/OfficeBearerCard.vue [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  data() {
+    return {
+      cardPage: 0
+    };
+  }
+};
+</script>
+
+<style scoped>
+.fh {
```
Comment (ajatprabha) [inline src/components/OfficeBearerCard.vue]: Perhaps a more meaningful class name?

File: src/components/OfficeBearerCard.vue (hunk 2 above)
Comment (killbotXD) [reply src/components/OfficeBearerCard.vue]: | 
does this O.O i didn't knew you can do this

File: src/views/OfficeBearers.vue [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<template lang="pug">
+  v-content.justify-center(justify-center)
+    v-card.text-center(class="blue text-center mx-auto" max-width="800").mt-4
+      v-card-text(class="white--text")
+        h1.font-weight-regular Office Bearer
+    v-container
+      v-row
+        v-col(cols="12" md="4" v-for="x in 3")
```
Comment (ajatprabha) [inline src/views/OfficeBearers.vue]: Add `:key="i"`

Comment (killbotXD) [reply src/views/OfficeBearers.vue]: you mean :key="x"?

**Pull request merged**